SHARED-CI.md

```diff
@@ -64,6 +64,25 @@ task                                    👈 all tasks go here
 
 ## ☑️ CI workflows
 
+### Kustomize Build
+
+- script: [`hack/build-manifests.sh`](hack/build-manifests.sh)
+  - Generates task manifest YAML files from Kustomize definitions (kustomize.yaml, patch.yaml)
+- script: [`hack/verify-manifests.sh`](hack/verify-manifests.sh)
+  - Verifies if all task manifests are up to date (no rebuild required).
+- workflow: [`.github/workflows/check-kustomize-build.yaml`](.github/workflows/check-kustomize-build.yaml)
+  - Checks if all task manifests are up to date (no rebuild required).
+
+With Kustomize, Task manifests are generated and kept consistent across the
+repository by composing base definitions (kustomize.yaml) with patches (patch.yaml).
```

Comment:
Would it be worth it to add an example into the `Expected repository structure` section?

Reply:
Not sure it is of great value, but I added it.

Reply:
dropped it.